crates/rolldown/src/stages/link_stage/bind_imports_and_exports.rs

```diff
@@ -465,7 +470,7 @@ impl LinkStage<'_> {
                 if cursor > 0 {
                   resolved_map.insert(
                     member_expr_ref.span,
-                    (Some(canonical_ref), member_expr_ref.props[cursor..].to_vec()),
```

Comment:
@IWANABETHATGUY I didn't turn off the feature. Let's give a try and see if it could survive